tests/src/main/java/net/neoforged/neoforge/debug/capabilities/handlers/energy/EnergyTestsSetup.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.debug.capabilities.handlers.energy;
+
+import com.mojang.serialization.Codec;
+import java.util.function.Supplier;
+import net.minecraft.core.component.DataComponentType;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.network.codec.ByteBufCodecs;
+import net.minecraft.world.item.Item;
+import net.neoforged.neoforge.capabilities.Capabilities;
+import net.neoforged.neoforge.capabilities.RegisterCapabilitiesEvent;
+import net.neoforged.neoforge.registries.DeferredItem;
+import net.neoforged.neoforge.registries.DeferredRegister;
+import net.neoforged.neoforge.transfer.handlers.templates.energy.EnergyBufferComponentHandler;
+import net.neoforged.testframework.TestFramework;
+import net.neoforged.testframework.annotation.OnInit;
+import net.neoforged.testframework.annotation.TestGroup;
+import net.neoforged.testframework.registration.DeferredItems;
+import net.neoforged.testframework.registration.RegistrationHelper;
+
+public class EnergyTestsSetup {
+    @TestGroup(name = "Energy Handler Group", enabledByDefault = true)
```

Comment:
Is... this annotation valid here? I haven't used the test groups but this certainly seems odd

Reply:
We discussed this in VC earlier, this is indeed the correct approach and the naming just tripped me up.